controllers/hibp.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const DB = require("../db/DB");
+
+
+async function notify (req, res) {
+  const hashes = req.body.hashSuffixes.map(suffix=>req.body.hashPrefix + suffix);
+
+  const subscribers = await DB.getSubscribersByHashes(hashes);
+
+  const reqBreachName = req.body.breachName.toLowerCase();
+  const breach = req.app.locals.breaches.filter(breach => breach.Name.toLowerCase() === reqBreachName)[0];
```

Comment:
Same as comment below, we _may_ want to use `.find()` here: https://github.com/mozilla/blurts-server/pull/263/files#r211107305

Also, because we're duplicating this code in 2 places in at least 2 files, we may want to have some shared `getFilterByName(name) {...}` convenience method.